### PR TITLE
feature/block-contents The blocks are complete

### PR DESCRIPTION
**Pull request by onlyabdiel**
Please check this bunch of components.

### CODE REVIEW

File: src/components/molecules/CardInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const CardInfo = () => {
```
Comment (WS-Jedp) [inline src/components/molecules/CardInfo.jsx]: It wil be better if we configure the props for send the necessary information.
```suggestion
const CardInfo = ({title, description}) => {
```

File: src/components/molecules/CardInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  return (
+    <div className="cardInfo">
+      <h2>Information</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam porta nisl non quam congue lacinia. Nullam ornare mi eu elit ultricies dapibus. Aenean congue sollicitudin ante at imperdiet. Nam ac sapien et ipsum sodales porta in id sapien. Phasellus eu nulla magna. Quisque orci ex, gravida quis rutrum nec.</p>
```
Comment (WS-Jedp) [inline src/components/molecules/CardInfo.jsx]: Same with props
```suggestion
      <h2>{ title || information }</h2>
      <p>{ description || lorem... }</p>
```

File: src/components/molecules/CuriosityFacts.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const CurosityFacts = () => {
```
Comment (WS-Jedp) [inline src/components/molecules/CuriosityFacts.jsx]: Same thing of the props
```suggestion
const CurosityFacts = ({ title, description }) => {
```

**Review by WS-Jedp — APPROVED** [review]
We can continue like this but it will be good fix these "problems"

File: src/components/molecules/CardInfo.jsx (hunk 1 above)
Comment (onlyabdiel) [reply src/components/molecules/CardInfo.jsx]: Done!

File: src/components/molecules/CardInfo.jsx (hunk 2 above)
Comment (onlyabdiel) [reply src/components/molecules/CardInfo.jsx]: Solved

File: src/components/molecules/CuriosityFacts.jsx (hunk 3 above)
Comment (onlyabdiel) [reply src/components/molecules/CuriosityFacts.jsx]: Ready

**Pull request merged**